### PR TITLE
Add quantisation to mark feature writer

### PR DESCRIPTION
**Pull request by simoncozens**
Reworking of #513 to use the existing mark feature writer.

### CODE REVIEW

File: Lib/ufo2ft/util.py [hunk 1]
```diff
@@ -481,3 +482,8 @@ def _loadPluginFromString(spec, moduleName, isValidFunc):
         raise ValueError("options have incorrect format: %r" % kwargs)
 
     return klass(**options)
+
+
+def quantize(number, degree):
```
Comment (anthrotype) [inline Lib/ufo2ft/util.py:487]: maybe rename `degree` parameter to `factor`?

**Review by anthrotype — APPROVED** [review]
lgtm, thanks!

**Comment by anthrotype** [comment]
I guess the kern feature writer as well could benefit from a similar `quantization` option

**Comment by simoncozens** [comment]
I wondered about that (which is why quantise is in utils, so it can be shared). I'll try it and see if it makes much difference to file size.

**Pull request merged**